portal/emailMessages.py

```diff
@@ -54,6 +54,15 @@ def emailVerificationNeededEmail(request, token):
     }
 
 
+def emailVerificationNeededEmailNew(request, token):
```

Comment:
I'd rather you stay consistent with your redesign process and copy the file as _new.py :)